More test cases for tag.pos_tag and util.date, util.time, util.wordtonum

bact opened the pull request:
- More test cases
- Add minus number support to `thaiword_to_num()`
- Add "เมื่อคืน" to `thaiword_to_date()`
- Add zero padding for `time_to_thaiword()`
- Add zero padding, space padding, no padding for `thai_strftime()`

May partly fixed #411

bact committed:
More test cases for better coverage + small improvements in date/time conversions

bact committed:
Simple test case for minus input to thaiword_to_num()

bact committed:
Add option to select padding in thai_time2time()

bact committed:
Test case for thai_strftime() - not supported format

bact committed:
Add test case: thai_strftime(): no padding, non-escaped %

bact committed:
Add support for padding options in thai_strftime()

pep8speaks commented:
Hello @bact! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:































There are currently no PEP 8 issues detected in this Pull Request. Cheers! :beers: 

##### Comment last updated at 2020-05-24 23:00:19 UTC

bact committed:
Add error messages to Error raised

bact committed:
Fix test cases

bact committed:
Add deprecation warnings and tests
Rename thai_time -> time_to_thaiword, thai_time2time -> thaiword_to_thai, thai_day2datetime -> thaiword_to_date

bact committed:
Fix assertWarns and value comparison

bact committed:
Add "เมื่อคืน" to thaiword_to_date()

bact committed:
thai_time2time and thai_day2datetime are not released yet, no need to do deprecation

bact committed:
Add imports in util.__init__.py

bact committed:
Removing trailing whitespaces

bact committed:
Removing trailing spaces

bact committed:
Fix newmm long test test case
Deal with ValueError in thai_strftime
Add more soundex test cases

bact committed:
Add _std_strftime() helper to normalize strftime() output in different platforms, also handling ValuError exception

bact committed:
Add test case of missing hour marker for thaiword_to_time()

bact committed:
Fix direct imports for numtoword/wordtonum

bact committed:
Zero-fill year with less than 4 and 2 digits (%Y %y %G %g)

bact committed:
Handle space padded case for no padding option
